LayoutTests/http/tests/webAPIStatistics/screen-functions-accessed-data-collection.html

```diff
@@ -29,7 +29,7 @@
         var availLeft = screen.availLeft;
         var availHeight = screen.availHeight;
         var availWidth = screen.availWidth;
-        testRunner.statisticsNotifyObserver();
+        await testRunner.statisticsNotifyObserver();
```

Comment:
This await doesn't look like it actually does anything? Same for many of the other tests changed in this patch.

Reply:
You're right, but many of them will be needed later when I remove installStatisticsDidModifyDataRecordsCallback and move the callback contents to after the await.  Also, adding the await is useful to make sure that if anyone adds things to these tests in the future it will have awaited the promise.